puppeteer_get_substrate_bearer/get_substrate_bearer_office.js

```diff
@@ -19,18 +19,35 @@ function delay(time) {
 
     const windowWidth = 1920;
     const windowHeight = 1080;
-    
-    // Launch the browser in full screen and incognito mode
-    const browser = await puppeteer.launch({
-        // Useful for debugging
-        headless: true, // Change to 'false' to see the browser actions for debugging
 
-        // In case you have issues, you can try to use the following flags (often issues can occur from specific Chrome instances or profiles, sometimes clean ones help)
-        // https://stackoverflow.com/questions/57623828/in-puppeteer-how-to-switch-to-chrome-window-from-default-profile-to-desired-prof/57662769#57662769
+    let browser;
+    try {
```

Comment:
@ofrin-zen 
1. IMO Let's just add a small explicit comment on why the try-catch clause is implemented like it is (to resolve the raised issues from repo and testing). Same for the teams version as well.
2. We can remove the duplicate comment on "In case you have issues"  as well maybe or put it a bit above to generally reference this section.